src/AggregateFunctions/AggregateFunctionFactory.cpp

```diff
@@ -1,5 +1,5 @@
 #include <AggregateFunctions/AggregateFunctionFactory.h>
-#include <AggregateFunctions/AggregateFunctionCombinatorFactory.h>
+#include <AggregateFunctions/combinators/AggregateFunctionCombinatorFactory.h>
```

Comment:
Just an idea, maybe it is better to use less nesting, i.e. put everything inside `AggregateFunctionsCombinators`? Or at least use `Combinators` (upper case first latter), like for most directories.

Or maybe include `Combinator` into file name, and do not introduce separate directory?

@amosbird what do you think?

Reply:
> Or maybe include Combinator into file name, and do not introduce separate directory?

The reason I didn't name `AggregateFunctionCombinatorMinMax.h` is because the function itself is not a combinator. The real combinator lives in `.cpp` file. That's why all other combinators don't have such names.

> Or at least use Combinators (upper case first latter), like for most directories.

I've also thought about this idea. I referenced 'Functions/array' for it. But you're right; we should choose to use the majority of styles.